ubuntu-bionic-18.04-cloudimg-console.log

```diff
@@ -0,0 +1,678 @@
+[    0.000000] Linux version 4.15.0-54-generic (buildd@lgw01-amd64-014) (gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)) #58-Ubuntu SMP Mon Jun 24 10:55:24 UTC 2019 (Ubuntu 4.15.0-54.58-generic 4.15.18)

```

Comment:
Should this file be ignored?

Reply:
Oh, whoops. Yeah I will ignore that file. That was the Vagrant log file.